learn/advanced/asynchronous_operations.md

```diff
@@ -140,110 +130,65 @@ Had the task failed, the response would have included a detailed `error` object:
-
-The code sample below filters all tasks canceled by the tasks with `uid`s `9` and `15`:
+When cancelling tasks, Meilisearch generates a new `taskCancellation` task. Use the `canceledBy` filter to view all tasks canceled by one or more `taskCancelation` task:
 
 <CodeSamples id="async_guide_canceled_by_1" />
 
-#### Filter by date
+### Filter by date
 
-You can filter tasks using `beforeXAt` and `afterXAt` with the `enqueuedAt`, `startedAt`, and `finishedAt` fields:
+You can filter tasks by their `enqueuedAt`, `startedAt`, and `finishedAt` fields. To do so, prepend either `before` or `after` to each field name:
 
 - `enqueuedAt` → `beforeEnqueuedAt` or `afterEnqueuedAt`
 - `startedAt` → `beforeStartedAt` or `afterStartedAt`
 - `finishedAt` →`beforeFinishedAt`  or `afterFinishedAt`
 
-This filter accepts the [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format. The following date syntaxes are valid:
+This filter accepts dates formatted according to [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt):
 
 - `YYYY-MM-DD`
 - `YYYY-MM-DDTHH:MM:SSZ`
 - `YYYY-MM-DDTHH:MM:SS+01:00`
 
 <CodeSamples id="async_guide_filter_by_date_1" />
 
-The above code sample will return all tasks `enqueued` **after** 11:49:53 am on 11 Oct 2020. It will start with the tasks `enqueued` at 11:49:54 am on 11 Oct 2020.
+The above code sample will return all tasks `enqueued` **after** 11:49:53:00 on 11 Oct 2020.
 
 ::: note
-Date filters are equivalent to `<` or `>` operations. At this time, there is no way to perform a `≤` or `≥` operations with a date filter.
+Date filters are equivalent to `<` or `>` operations and do not include the specified value. It is not possible to perform `≤` or `≥` operations with a task date filter.
 :::
 
-#### Combine filters
+### Combine filters
 
-You can combine the above mentioned filters to get tasks meeting specific requirements. Use the ampersand character (`&`) to combine filters, equivalent to a logical `AND`. Use the comma character (`,`) to add multiple filter values for a single field, equivalent to a logical `OR`.
+You can combine task filters to get tasks meeting specific requirements. Use the ampersand character (`&`) to combine filters, equivalent to a logical `AND`.
 
-The following code sample will return all `documentAdditionOrUpdate` and `documentDeletion` type tasks with `processing` status in the `movies` index:
+The following code sample returns all `documentAdditionOrUpdate` and `documentDeletion` type tasks with `processing` status in the `movies` index:
 
 <CodeSamples id="async_guide_multiple_filters_1" />
 
-At this time, `OR` operations between different filters are not supported. For example, you cannot view only tasks which have a type of `documentAddition` **or** a status of `failed`.
+**`OR` operations between different filters are not supported.** For example, you cannot view only tasks which have a type of `documentAddition` **or** a status of `failed`.
 
-### Paginating tasks
+## Paginating tasks
```

Comment:
Very open to renaming this, since pagination mostly makes you think about front-end stuff. When people search "paginating", we want them to find the pagination guide, not this.